third_party/move/tools/move-coverage/src/source_coverage.rs

```diff
@@ -42,6 +46,98 @@ pub enum AbstractSegment {
     BoundedLeft { start: u32 },
 }
 
+#[derive(ValueEnum, Clone, Debug, Serialize)]
+pub enum ColorChoice {
```

Comment:
Not necessary do it in this PR if you are not comfortable, but you for enums like this where you want to convert back and forth to string, `strum` macro works nicely (https://docs.rs/strum_macros/latest/strum_macros/derive.EnumString.html). 

An example use can be seen here: https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/move-compiler-v2/src/lint_common.rs#L18

Reply:
Not in this PR.